### PR TITLE
refactor partial docs

### PR DESCRIPTION
**Pull request by devmanhinton**
@yaymukund 

Makes partials much shorter and clearer

### CODE REVIEW

**Comment by devmanhinton** [comment]
@bglusman

**Comment by bglusman** [comment]
@devmanhinton think this is OK without any examples for partials?  why not have examples?

**Comment by yaymukund** [comment]
Can you show a full example from loading a partial to using that partial?

And instead of `myPartial`, can you use a real example? Even something simple like:

``` html
<script type="text/enddash" name="superhero-navigation">
...
</script>

<div src="superhero-navigation"></div>
```

I do like the instinct to trim it down though. If your goal is to show how partials work, you don't need a collection with three things in it.

**Comment by devmanhinton** [comment]
@bglusman I did just because templates & their syntax is loaded elsewhere.

@yaymukund I can add that. We document loading templates elsewhere (or rather there is the start of that code in a PR. After that merged I think I need to document boostrap().). I can add it here as well though?

**Comment by yaymukund** [comment]
@devmanhinton I suggested it because people might not realize that loading partials is the same as loading templates...

**Comment by devmanhinton** [comment]
@yaymukund Cool. No worries working on an edit to the affect right now.

**Comment by devmanhinton** [comment]
@bglusman @yaymukund Good to check out again.

**Comment by devmanhinton** [comment]
@bglusman

File: README.md [hunk 1]
```diff
@@ -594,52 +594,59 @@ $('.content').html(boundTemplate.el);
...
-var person1 = new Backbone.Model({firstName: 'Tony', lastName: 'Stark', alias: 'IronMan'});
-var person2 = new Backbone.Model({firstName: 'James', lastName: 'Rhodes', alias: 'WarMachine'});
-var person3 = new Backbone.Model({firstName: 'Pepper', lastName: 'Potts', alias: 'none' });
-
-var people = new Backbone.Collection([person1, person2, person3]);
+```html
+<script type='text/enddash' name='superheroes'>
+  <img src="#{logo}" />
+  <div class='heroes-'>
+    <div src='superhero-navigation' data-replace></div>
```
Comment (yaymukund) [inline README.md:604]: Please use double quotes for HTML.

**Pull request merged**